### ISSUE TITLE
Do not hardcode css

### ISSUE THREAD

**Issue by maciejmiklas** [vankop/jsoneditor-react#41]
There is hardcoded css import:
`import 'jsoneditor/dist/jsoneditor.css';`

This makes any css customisation harder. Please remove this import so that we can import css of our choice.

**Comment by vaultec81** [comment]
I agree! I have found a work around that seems to work for anyone wondering. Passing CSS through `htmlElementProps` seems to work. But it isn't the best solution obviously.

**Comment by PunKHS** [comment]
Maybe someone has workaround? I would like to change the `.jsoneditor-menu { background-color: #4b4b4b }` and cannot do it through `htmlElementProps`

**Comment by vaultec81** [comment]
You might be able to go into the CSS code in the `node_modules` folder, or change the properties of the jsoneditor-menu class through javascript if that's possible.

**Comment by SeeThruHead** [comment]
can't import any css file in node_modules in next js

**Comment by SimonKlausLudwig** [comment]
would be great, this adds a ton of css and makes lazy loading of the css really hard.

**Comment by mr-bjerre** [comment]
@SeeThruHead I have the same issue. Found a solution?

**Comment by SeeThruHead** [comment]
no we didn't use this because of the issue

**Comment by SimonKlausLudwig** [comment]
It should be that hard to exclude the css file right ? Please take this into account..

**Comment by KirillSuhodolov** [comment]
is it possible to exclude css for next.js apps?

**Comment by anavgagneja** [comment]
@SeeThruHead did you find an alternative you liked that works with next js?

**Comment by amiram** [comment]
@anavgagneja this may help https://github.com/tsukhu/jsoneditor-react-ext
